modules/aws/s3.go

```diff
@@ -368,6 +446,29 @@ func AssertS3BucketPolicyExistsE(t testing.TestingT, region string, bucketName s
 	return nil
 }
 
+// AssertTargetLoggingBucketIsReturned checks if the given S3 bucket has logging enabled and a target bucket set.
+func AssertTargetLoggingBucketIsReturned(t testing.TestingT, region string, sourceBucketName string, logsTargetBucketName string) {
+	err := AssertTargetLoggingBucketIsReturnedE(t, region, sourceBucketName, logsTargetBucketName)
+	require.NoError(t, err)
+}
+
+// AssertTargetLoggingBucketIsReturned checks if the given S3 bucket has logging enabled and a target bucket set; returns error if not.
+func AssertTargetLoggingBucketIsReturnedE(t testing.TestingT, region string, sourceBucketName string, logsTargetBucketName string) error {
+	targetBucket, err := GetS3BucketLoggingTargetE(t, region, sourceBucketName)
+	if err != nil {
+		return err
+	}
+
+	if targetBucket == "" {
+		return err
+	}
+
+	if targetBucket == logsTargetBucketName {
+		return nil
+	}
+	return err
+}
```

Comment:
This will return `nil` because `err` is `nil` at this point (it failes the first if condition, which checks for non-nil). 

That said, this is a relatively simple function on top of `GetS3BucketLoggingTarget`, so I think it fails the complexity litmus test for inclusion in terratest.

Instead, you can implement this inline in your test as:

```
targetBucket := aws.GetS3BucketLoggingTarget(...)
assert.Equal(t, targetBucket, logsTargetBucketName)
```

Reply:
sweet! that simplifies it a lot.